lib/src/main/java/com/diffplug/spotless/json/JsonFormatterStep.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.json;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.Objects;
+
+import com.diffplug.spotless.FormatterFunc;
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.JarState;
+import com.diffplug.spotless.Provisioner;
+
+public final class JsonFormatterStep {
+	private static final String MAVEN_COORDINATE = "org.json:json:";
+	private static final String DEFAULT_VERSION = "20210307";
```

Comment:
:question: should we be making this configurable so folks can pull a later version, or are we happy requiring a bump of Spotless?

Reply:
JSON is such an old and solid standard, it seems very unlikely that users will ever care about which version of `org.json` is being used.